vendor.conf

```diff
@@ -8,7 +8,7 @@ github.com/containerd/continuity                    efbc4488d8fe1bdc16bde3b2d299
 github.com/containerd/fifo                          f15a3290365b9d2627d189e619ab4008e0069caf
 github.com/containerd/go-runc                       7016d3ce2328dd2cb1192b2076ebd565c4e8df0c
 github.com/containerd/nri                           0afc7f031eaf9c7d9c1a381b7ab5462e89c998fc
-github.com/containerd/ttrpc                         v1.0.1
+github.com/containerd/ttrpc                         079556c0a5f85bed5aeeaaf9e9e298af09b30c6c
```

Comment:
Can we tag a new release for this one? https://github.com/containerd/ttrpc/compare/v1.0.1...master

Reply:
We should tag this

Reply:
make senses. I was thinking we should tag it before major release because before next major release, the ttrpc might be updated. 

will do it.